pylint/checkers/base.py

```diff
@@ -1490,14 +1490,14 @@ def _check_unreachable(self, node):
             self.add_message("unreachable", node=unreach_stmt)
 
     def _check_not_in_finally(self, node, node_name, breaker_classes=()):
-        """check that a node is not inside a finally clause of a
-        try...finally statement.
-        If we found before a try...finally block a parent which its type is
+        """check that a node is not inside a 'finally' clause of a
+        'try...finally' statement.
+        If we found before a 'try...finally' block a parent which its type is
```

Comment:
This sentences doesn't work.

Reply:
Right. I did not read it, just added `'`. guess we have to fix it now.
```
        If we found a parent which type is in breaker_classes before
        a 'try...finally' block we skip the whole check.
```
?

Reply:
Seems good to me!